instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/internal/ContextPropagationDebug.java

```diff
@@ -68,6 +69,7 @@ public static void debugContextLeakIfEnabled() {
     }
   }
 
+  @Nullable
```

Comment:
Is it possible to return an empty list instead of null?

Reply:
It is, but it'd result in a bit more code being added - and since it's a private, internal utility method I think it's fine to keep `@Nullable` here.